entrypoint.sh

```diff
@@ -11,6 +11,17 @@ function appendParams(){
 
 kn_command=("kn" "service")
 
+namespace=""
+
+if [[ -n $INPUT_SERVICE_NAMESPACE ]]; then
+    echo "Setting service namespace to '$INPUT_SERVICE_NAMESPACE'"
+    namespace="--namespace=$INPUT_SERVICE_NAMESPACE"
+else
+    echo "No namespace provided"
+fi
+
+echo $namespace
```

Comment:
Why echo it here? Either we echoed it above, or it is empty

Reply:
It was just for testing, forgot to remove that.